src/sage/doctest/forker.py

```diff
@@ -931,7 +936,7 @@ def summarize(self, verbose=None):
             sage: from sage.doctest.control import DocTestDefaults; DD = DocTestDefaults()
             sage: import doctest, sys, os
             sage: DTR = SageDocTestRunner(SageOutputChecker(), verbose=False, sage_options=DD, optionflags=doctest.NORMALIZE_WHITESPACE|doctest.ELLIPSIS)
-            sage: DTR._name2ft['sage.doctest.forker'] = (1,120)
+            sage: DTR._stats['sage.doctest.forker'] = (1,120)
```

Comment:
```suggestion
            sage: DTR._stats['sage.doctest.forker'] = (1,120,0)
```

Reply:
Yes, that's the point.

I think the doctest can be left as is because it's just mocking the result here… otherwise you'll need to do a version check.